platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -222,7 +305,11 @@ def get_presence(self):
         return eeprom_raw is not None
 
     # Read out any bytes from any offset
-    def read_eeprom(self, offset, num_bytes):
+    def _read_eeprom_specific_bytes(self, offset, num_bytes):
```

Comment:
Not critical, but I suggest renaming this function to _read_eeprom_sfp_bytes or a name you choose that tells what function is actually doing.